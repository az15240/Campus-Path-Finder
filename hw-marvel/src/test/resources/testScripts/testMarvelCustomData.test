# Tests marvel.csv custom data. Data is adapted from https://edstem.org/us/courses/32028/discussion/2592476

LoadGraph marvel marvel.csv

FindPath marvel IRON-MAN/TONY-STARK KANGAROO-II

FindPath marvel GOOM MR.-FANTASTIC/REED-R

FindPath marvel SHA-SHAN SQUIRREL-GIRL

FindPath marvel GOOM HOFFMAN