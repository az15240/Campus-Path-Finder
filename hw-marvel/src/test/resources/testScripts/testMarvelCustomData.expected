# Tests marvel.csv custom data. Data is adapted from https://edstem.org/us/courses/32028/discussion/2592476

loaded graph marvel

path from IRON-MAN/TONY-STARK to KANGAROO-II:
IRON-MAN/TONY-STARK to ABSORBING-MAN/CARL-C via A-183
ABSORBING-MAN/CARL-C to KANGAROO-II via W2-164

path from GOOM to MR.-FANTASTIC/REED-R:
GOOM to VENUS/APHRODITE/VICT via M/VS-1/3
VENUS/APHRODITE/VICT to ANGEL/WARREN-KENNETH via CHAMP-1
ANGEL/WARREN-KENNETH to MR.-FANTASTIC/REED-R via A-3

path from SHA-SHAN to SQUIRREL-GIRL:
SHA-SHAN to ANGEL/WARREN-KENNETH via PPTSS-18
ANGEL/WARREN-KENNETH to DR.-DOOM/VICTOR-VON via CHAMP-16
DR.-DOOM/VICTOR-VON to SQUIRREL-GIRL via M/SH3-8/3

path from GOOM to HOFFMAN:
no path found
